lib/active_merchant/billing/gateways/stripe.rb

```diff
@@ -283,7 +284,8 @@ def add_destination(post, options)
       end
 
       def add_expand_parameters(post, options)
-        post[:expand] = Array.wrap(options[:expand])
+        post[:expand] ||= []
+        post[:expand].concat(Array.wrap(options[:expand]).map(&:to_sym)).uniq!
```

Comment:
I see, `Array.wrap` is a tricky one. This comment is not in the scope of this PR anyway, so :+1: